docs/sampling/README.md

```diff
@@ -0,0 +1,262 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Sampling
+--->
+
+# Semantic Conventions for Sampling
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- toc -->
+
+- [Probability sampling](#probability-sampling)
+  - [Overriding sampling decisions](#overriding-sampling-decisions)
+  - [Overriding sampling randomness](#overriding-sampling-randomness)
+  - [Sampling threshold](#sampling-threshold)
+  - [Sampling randomness](#sampling-randomness)
+  - [No definition for Scope and Resource attributes](#no-definition-for-scope-and-resource-attributes)
+- [Span sampling attributes](#span-sampling-attributes)
+- [Logs sampling attributes](#logs-sampling-attributes)
+- [Examples](#examples)
+  - [Head sampling](#head-sampling)
+  - [Tail sampling](#tail-sampling)
+
+<!-- tocstop -->
+
+These attributes reflect the effect of sampling in a telemetry
+collection pipeline.  These attributes describe how items of telemetry
+were collected, making it possible for Span-to-Metrics pipelines
+and Logs-to-Metrics pipelines to accurately count Spans and Log
+Records of telemetry, before sampling, in a probabilistic sense.
+
+These attributes MAY be modified by components in a collection
+pipeline to convey successive sampling that has been carried out for a
+particular item of telemetry, using the conventions for consistent
+sampling described here.  In that sense, telemetry consumers
+should see these attributes as telemetry metadata.
+
+## Probability sampling
+
+The OpenTelemetry sampling decision is defined in terms of a Threshold
+value and a Randomness value, each containing 56 bits of information.
+
+A constant known as _maximum adjusted count_ (`MaxAdjustedCount`),
```

Comment:
It could be just me, but I think that `max`-_something_ suggests inclusiveness, so this can be confusing. How about `AdjustedCountLimit`?

Reply:
I am fine with `MaxAdjustedCount`. It is inclusive with respect to the adjusted count. However, I understand that it can be a little confusing as it is also used as an exclusive upper limit for the threshold and the random value.